commercetools-models/src/test/java/io/sphere/sdk/client/UserAgentIntegrationTest.java

```diff
@@ -28,7 +28,7 @@ public void userAgent() throws Exception {
         }
     }
 
-    private static class FakeHttpClient implements HttpClient {
+    public static class FakeHttpClient implements HttpClient {
```

Comment:
Is this really related to this issue?

Reply:
No but it caused some access problems with OSGi so i decided to make it public, just as a shortcut to fix the problem.

Reply:
Ok and thanks for your explanation 👍